common/meta/types/proto/request.proto

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 Datafuse Labs.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package meta;
+
+message SeqV {
+  uint64 seq = 1;
+  bytes data = 2;
+}
+
+message TxnKey {
+  string key = 1;
+  string tenant = 2;
+  string tree = 3;
+}
```

Comment:
is it possible that we may introduce some features like tenant isolation inside the metasrv in the future? if so, a metadata about tenent inside each key maybe helpful. 

likewise we have an JWT inside the user session, each JWT is associated with a limitted scope of JWT, we may pass the JWT alongside to the metasrv, and metasrv can restrict the session can only access the data inside one tenant.

Reply:
>is it possible that we may introduce some features like tenant isolation inside the metasrv in the future? if so, a metadata about tenent inside each key maybe helpful.

It can be done. But if to add this feature is still up to the design principle:
Whether to keep the metasrv a pure kv service like etcd or it has to understand some of the business logic.

Currently `tenant` is embedded into the `key` thus metasrv do not need to understand `tenant`. Are we gonna break this rule?